packages/gatsby-theme-newrelic/src/components/Link.js

```diff
@@ -81,13 +83,18 @@ const Link = forwardRef(
       );
     }
 
+    if (className === 'gatsby-resp-image-link') {
```

Comment:
[nit] If you didn't want to destructure + add the `className` to everything, you could also do this trick:

```js
if (props.className === 'gatsby-resp-image-link')
```

then leave everything else as it was.

Reply:
You might also consider wrapping this in a helper function to make it clear what you're wanting to do here.

```js
// or whatever makes sense to pass in here
if (isImageLink({ ...props, to }))
```

Reply:
i initially used `props.className` but got prop type validation linting error 🤷 

good call on moving to function like the others

Reply:
I think if you add the `propType` itself, that should resolve that eslint warning without needing to destructure the prop in the component. I forgot about that rule 😆

Reply:
oh didn't realize i could just add `className` to `Link.propTypes`. it passed the linting check and classes are applied to other links as expected (tested change in local docs repo as well).